### DISCUSSION TITLE
Bug: display of the number of packages

### DISCUSSION THREAD

**Discussion by deadblackclover** [community/community#8143] [Actions]
My profile is not displaying package counts correctly. At the moment, all packages have been removed.

There is one package here, but the packages are empty.
![image](https://user-images.githubusercontent.com/26848022/143611864-6a332876-a816-471f-b27d-2cfc1803d423.png)

There are two.
![image](https://user-images.githubusercontent.com/26848022/143611905-33654b04-ed05-4bc5-b521-3d1bc024d340.png)

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
